apps/list.c

```diff
@@ -513,6 +616,23 @@ int list_main(int argc, char **argv)
     char *prog;
     HELPLIST_CHOICE o;
     int one = 0, done = 0;
+    struct {
+        unsigned int commands:1;
+        unsigned int digest_commands:1;
+        unsigned int digest_algorithms:1;
+        unsigned int mac_algorithms:1;
+        unsigned int cipher_commands:1;
+        unsigned int cipher_algorithms:1;
+        unsigned int pk_algorithms:1;
+        unsigned int pk_method:1;
+        unsigned int engines:1;
+        unsigned int disabled:1;
+        unsigned int missing_help:1;
+        unsigned int objects:1;
+        unsigned int options:1;
+    } todo = { 0, };
```

Comment:
Not overally impressed with this variable name :)

Reply:
Feel free to suggest something else

Reply:
flags, features ?

Reply:
[ahem] I'm not terribly impressed with those names :wink:

Reply:
todo3.0 ?

Reply:
Er...  you jest, I'm sure :wink: